src/screens/Roles/Roles.tsx

```diff
@@ -10,11 +10,47 @@ import {
   ORGANIZATION_CONNECTION_LIST,
   USER_LIST,
 } from 'GraphQl/Queries/Queries';
-import { UPDATE_USERTYPE_MUTATION } from 'GraphQl/Mutations/mutations';
+import {
+  UPDATE_USERTYPE_MUTATION,
+  ADD_ADMIN_MUTATION,
+  REMOVE_ADMIN_MUTATION,
+  ADD_MEMBER_MUTATION,
+  REMOVE_MEMBER_MUTATION,
+} from 'GraphQl/Mutations/mutations';
 import PaginationList from 'components/PaginationList/PaginationList';
 import NotFound from 'components/NotFound/NotFound';
 import { errorHandler } from 'utils/errorHandler';
 
+interface User {
+  __typename: string;
+  _id: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+}
+
+interface Organization {
+  __typename: string;
+  _id: string;
+  name: string;
+  admins: User[];
+  members: User[];
+}
+
+interface Data {
```

Comment:
Why opt to use interfaces here instead of types?

Reply:
Interfaces and types share many similarities in functionality and purpose. If you prefer to use types over interfaces, I can modify the code to utilize types instead.

Reply:
In this case I think it would be better to use types. Interfaces can be extended by declaring it a second time.

Reply:
Yes I have updated them to types